gyb_syntax_support/Token.py

```diff
@@ -11,7 +11,7 @@ class Token(object):
     def __init__(self, name, kind, name_for_diagnostics,
                  unprefixed_kind=None, text=None, classification='None',
                  is_keyword=False, requires_leading_space=False,
-                 requires_trailing_space=False):
+                 requires_trailing_space=False, associated_value_class=None):
```

Comment:
Should we also add this to the swift child file

Reply:
I’ll add it in a follow-up PR that also starts generating Keywords.swift using CodeGeneration: https://github.com/apple/swift-syntax/pull/1174